### PR TITLE
Added back button property and her event to allow work backButton.

### PR DESCRIPTION
**Pull request by pablovargan**

### CODE REVIEW

File: CSharp/Project/MVVMtpl/MVVMtpl.Shared/ViewModels/ExampleViewModel.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public ExampleViewModel(NavigationService navigationService, MessagingService me
             this.navigationService = navigationService;
             this.messagingService = messagingService;
 
-            messagingService.Register();
+            //messagingService.Register();
```
Comment (danielrozo) [inline CSharp/Project/MVVMtpl/MVVMtpl.Shared/ViewModels/ExampleViewModel.cs]: I'd be reluctant to have this commented-out (this is the main reason we have a git, so we don't have this kind of things). If it's trully not necessary, just delete it, if it was a test, please reset it back to normal. Nevertheless, please check with @CKGrafico to decide if it's unnecessary, or why would you delete it.

File: CSharp/Project/MVVMtpl/MVVMtpl.Shared/App.xaml.cs [hunk 2]
```diff
@@ -133,5 +137,22 @@ private void OnSuspending(object sender, SuspendingEventArgs e)
...
         }
+
+#if WINDOWS_PHONE_APP
+        /// <summary>
+        /// Invoked when back button is pressed.
+        /// </summary>
+        /// <param name="sender">The source of the suspend request.</param>
+        /// <param name="e">Details about the suspend request.</param>
+        private void HardwareButtons_BackPressed(object sender, Windows.Phone.UI.Input.BackPressedEventArgs e)
+        {
```
Comment (danielrozo) [inline CSharp/Project/MVVMtpl/MVVMtpl.Shared/App.xaml.cs]: Correct me if I'm wrong, or perhaps I'm missing something, but I woud say this is the default behaviour for the WP apps.

**Comment by pablovargan** [comment]
I forgot uncomment this line before making the commit! Sorry.

File: CSharp/Project/MVVMtpl/MVVMtpl.Shared/ViewModels/ExampleViewModel.cs (hunk 1 above)
Comment (pablovargan) [inline CSharp/Project/MVVMtpl/MVVMtpl.Shared/ViewModels/ExampleViewModel.cs]: I forgot uncomment this line before making the commit! Sorry.

File: CSharp/Project/MVVMtpl/MVVMtpl.Shared/App.xaml.cs (hunk 2 above)
Comment (mapaux) [inline CSharp/Project/MVVMtpl/MVVMtpl.Shared/App.xaml.cs]: It's necessary because it's the correct way to do that

**Pull request merged**